github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
Potential Bug: The version of an indirect dependency "golang.org/x/sys" has been changed from v0.9.0 to v0.8.0. Please verify that this will not cause any compatibility or functionality issues.